docs/graphql/manual/queries/query-filters.rst

```diff
@@ -187,6 +187,35 @@ Fetch a list of articles that were published on a certain date (``published_on``
       }
     }
 
+Example: Column
+^^^^^^^^^^^^^^^^^^^^^^^
+Fetch a list of ``price`` *(a column)* rows that equal the configured value for the category ``max_category_price`` *(a column in the same table)*:
```

Comment:
Can we actually return a list (two products), so that it's clear that it's not just one?

Reply:
you bet